src/service/url-impl.js

```diff
@@ -146,21 +146,18 @@ export class Url {
    * @param {string} resourceUrl The URL of the document to load
    * @return {string}
    */
-  getCdnUrlOnCdnOrigin(currentLocation, resourceUrl) {
-    if (!isProxyOrigin(currentLocation)) {
+  getCdnUrlOnOrigin(currentLocation, resourceUrl) {
```

Comment:
I don't think passing two URLs makes sense. We should just pass one, and transform it as necessary.